src/Modules/AuthModule/components/ResetPass/ResetPass.tsx

```diff
@@ -1,7 +1,252 @@
+                                {...register("password", {
+                                  required: "password is required ",
+                                  pattern: {
+                                    value:
+                                      /^(?=.*\d)(?=.*[a-z])(?=.*[A-Z])(?=.*[a-zA-Z]).{8,}$/,
+                                    message:
+                                      "Password must contain at least 8 characters, including upper and lowercase letters, and numbers",
+                                  },
+                                })}
+                              />
+                              <label
+                                htmlFor="input-field"
+                                className="input-label"
+                              >
+                                Password
+                              </label>
+                              <span className="input-highlight"></span>
+                            </div>
+
+                            
+
+                            <i
+                              role="button"
+                              onClick={togglePasswordVisibility}
+                              className={`icon fa-regular  ${
+                                showPassword ? "fa-eye" : "fa-eye-slash"
+                              }`}
+                            ></i>
+                          </div>
+                          {errors.password && (
+                            <p className="text-start text-danger ps-3">
+                              {errors.password.message}
+                            </p>
+                          )}
+
+<div className="group-input ">
+                            <div className="input-container  ">
+                              <input
+                                placeholder="Confirm your new password"
+                                className={`input-field ${
+                                  errors.password && "border-danger "
+                                }`}
+                                type={showPassword ? "text" : "password"}
+                                {...register("confirmPassword", {
+                                  required: "confirm password is required ",
+                                  validate: validateConfirmPassword
+                                })}
+                              />
+                              <label
+                                htmlFor="input-field"
+                                className="input-label"
+                              >
+                                Confirm Password
+                              </label>
+                              <span className="input-highlight"></span>
+                            </div>
+
+                            
+
+                            <i
+                              role="button"
+                              onClick={toggleConfirmPasswordVisibility}
```

Comment:
We shouldn't be giving onClick to icons, we can make a button and put the icon in it and the button will be given the onClick. This is better for accessibility, as we will be able to access the button using the tap key, we will also be able to perform the action using the space or enter.